### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by EmIsGreat**
Update Readme to include instructions on how to add MineColonies as a dependency to a forge mod.

# Changes proposed in this pull request:
-Update Readme

Review please

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,24 @@ MineColonies crashes every time? Have a suggestion? Found a bug? Create an issue
 
 ## For Developers
 
+### Adding MineColonies as Maven Dependecy
```
Comment (Thodor12) [inline README.md]: Maven?

**Review by Thodor12 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (EmIsGreat) [reply README.md]: As in "link" to the maven repo. But I could change it to something else if you want to.

**Review by Raycoms — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (Thodor12) [reply README.md]: I meant more as in, you are saying "Maven" dependency, but you are showing Gradle code.

Also "Dependency" is misspelled I just noticed.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,24 @@ MineColonies crashes every time? Have a suggestion? Found a bug? Create an issue
...
+
+In Forge, add the following to your `build.gradle`:
+
+```gradle
+repositories {
+    	maven { url 'https://ldtteam.jfrog.io/ldtteam/modding' }
+}
+
+dependencies {
+    	implementation fg.deobf("com.ldtteam:minecolonies:${minecolonies_version}")
```
Comment (marchermans) [inline README.md]: Please fix the indent

File: README.md (hunk 1 above)
Comment (marchermans) [reply README.md]: There is no other kind of dependency
So please remove the "Maven" from the statement.

**Review by marchermans — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (EmIsGreat) [reply README.md]: > Please fix the indent

I changed it to 4 spaces now but if you meant another indent please let me know

**Pull request merged**